### ISSUE TITLE
Kernel#puts is not working

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#17]
Kernel#puts is swallowed by Shoes and not displayed in the Shoes console. Windows newest cshoes.exe will however display puts accordingly to the calling console (cmd) but Shoes will not on MacOS X. This issue on Linux is currently undetermined.

```ruby
Shoes.app(:title => "Copy and Paste", :width => 450, :height => 250) {
   para "Check Shoes console (alt-/)"
   puts "This should be displayed in Shoes console."
}
```

**TEMPORARY WORKAROUND**
```ruby
module Kernel
   def puts(str)
      info str
   end
end

Shoes.app(:title => "Copy and Paste", :width => 450, :height => 250) {
   para "Check Shoes console (alt-/)"
   puts "This should be displayed in Shoes console."
}
```

**Comment by passenger94** [comment]
doesn't work on ubuntu either but we have  **Shoes.p**
```ruby
Shoes.app(:title => "Copy and Paste", :width => 450, :height => 250) {
   para "Check Shoes console (alt-/)
   Shoes.p "This should be displayed in Shoes console."
}
```

**Comment by ccoupe** [comment]
On OSX, the naked puts goes to the System Console (open Console.app if you want to watch the collective stdout/stderr stream).  @passenger94: The Linux equivalent  to OSX behavior would be to send stdout/stderr from Shoes to syslog, IFF Shoes was started from a Desktop Menu

@BackOrder _If_ you can get Shoes started without using the 'open' command on OSX then stdout/stderr will appear on the initiating terminal.

**Comment by ccoupe** [comment]
Fixed (differently) in 3.2.24
